include/circt/Dialect/Arc/ArcPasses.td

```diff
@@ -62,6 +62,16 @@ def FindInitialVectors : Pass<"arc-find-initial-vectors", "mlir::ModuleOp"> {
   let summary = "Find initial groups of vectorizable ops";
   let constructor = "circt::arc::createFindInitialVectorsPass()";
   let dependentDialects = ["arc::ArcDialect"];
+  let statistics = [
+    Statistic<"numOfVectorizedOps", "vectorOps",
```

Comment:
Nit: the name `vectorOps` suggests the same as `numOfVectorsCreated`, i'd rename it slightly:
```suggestion
    Statistic<"numOfVectorizedOps", "vectorizedOps",
```